### PR TITLE
[EOS-22923] : Auth IEM basic template structure for Auth server

### PR DESCRIPTION
**Pull request by pranavPawar07**
- This will add basic template/class structure of Auth IEM alerts.
- this will provide basic API for calling and sending IEM alerts from authserver.
- Integration of Auth with new IEM alerts is not part of this PR. it will be done in next PI.

Signed-off-by: Pranav Diliprao Pawar <pranav.pawar@seagate.com>

### CODE REVIEW

File: auth/server/src/main/java/com/seagates3/dao/ldap/LDAPUtils.java [hunk 1]
```diff
@@ -353,11 +373,15 @@ static void bind(String dn, String password) throws LDAPException {
                        ldapException.getCause() + ". Message: " +
                        ldapException.getMessage());
           LOGGER.debug("Stacktrace: " + ldapException);
-          IEMUtil.log(
-              IEMUtil.Level.ERROR, IEMUtil.LDAP_EX,
-              "An error occurred while establishing ldap connection. For " +
-                  "more " + "information, please refer Troubleshooting Guide.",
-              String.format("\"cause\": \"%s\"", ldapException.getCause()));
+        //   IEMUtil.log(
```
Comment (nileshgovande) [inline auth/server/src/main/java/com/seagates3/dao/ldap/LDAPUtils.java:376]: Migration is not supposed to be part of this patch.

Comment (sachitanands) [reply auth/server/src/main/java/com/seagates3/dao/ldap/LDAPUtils.java]: commented this new code and reverts the original code for IEM alert.

**Review by abhilekh268 — APPROVED** [review]

File: auth/server/src/main/java/com/seagates3/controller/AccountController.java [hunk 2]
```diff
@@ -44,12 +42,10 @@
 import com.seagates3.s3service.S3AccountNotifier;
```
Comment (shalakadharap) [inline auth/server/src/main/java/com/seagates3/controller/AccountController.java:42]: If no changes in file then remove the file from commit.

File: auth/server/src/main/java/com/seagates3/model/AuthIEMAlert.java [hunk 3]
```diff
@@ -1,6 +1,73 @@
...
+
 public
 class AuthIEMAlert {
-  // TODO Base template class for IEM alerts
+
+ private
+ final static Logger LOGGER =
+     LoggerFactory.getLogger(AuthIEMAlert.class.getName());
+ public
+  static final String LDAP_EX = "0040010001";
```
Comment (shalakadharap) [inline auth/server/src/main/java/com/seagates3/model/AuthIEMAlert.java:19]: Why we repeating these code definitions here?
Cant we use the same from IEMUtil.java

Comment (sachitanands) [reply auth/server/src/main/java/com/seagates3/model/AuthIEMAlert.java:19]: had a discussion with Basav earlier and we decided to keep this separate

File: auth/server/src/main/java/com/seagates3/controller/AccountController.java (hunk 2 above)
Comment (sachitanands) [reply auth/server/src/main/java/com/seagates3/controller/AccountController.java]: done.. other changes are there which are of removing unused packages warnings and code-formatter fixes

File: auth/server/src/main/java/com/seagates3/model/AuthIEMAlert.java (hunk 3 above)
Comment (shalakadharap) [reply auth/server/src/main/java/com/seagates3/model/AuthIEMAlert.java:19]: @bkirunge7

Comment (shalakadharap) [reply auth/server/src/main/java/com/seagates3/model/AuthIEMAlert.java:19]: ok approving.

**Review by shalakadharap — APPROVED** [review]

**Comment by nileshgovande** [comment]
All jobs pass - merging

**Pull request merged**